Add documentation for the `govuk-visually-hidden` utility classes (#3061)
alphagov/govuk-design-system: src/styles/layout/index.md

- `govuk-!-display-none` to remove the element from the page

You can also remove elements from the printed version of the page using `govuk-!-display-none-print`.

> ## Hide elements whilst keeping them accessible to screen readers

Reviewer: ```suggestion
## Hide elements and keep them accessible to screen readers
```

> You can stop elements from being displayed on the page whilst keeping their contents available to screen readers using `govuk-visually-hidden`. This can be used to present content to screen reader users that sighted users don't need.

Reviewer: ```suggestion
You can stop elements from being displayed on the page and keep their contents available to screen readers using `govuk-visually-hidden`. You can use this to present content to screen reader users that sighted users do not need.
```
